lib/libzstd/Makefile.am

```diff
@@ -6,10 +6,6 @@ VPATH = $(top_srcdir)/module/zstd
 # Set it for other compilers, too.
 AM_CFLAGS += -fno-tree-vectorize
 
-# SSE register return with SSE disabled if -march=znverX is passed
-AM_CFLAGS += -U__BMI__
-
-
```

Comment:
removing this will lead to compile errors on some X64 architectures, depending on your -march option. i added this workaround since i was running into that issue. i know that the origin of that problem is a upstream toolchain bug which might be resolved with newer compiler versions. but we should consider all supported toolchains

Reply:
Sure, that being in this rev was just an oversight, I had removed it at one point and didn't revert it before pushing.